src/components/ButtonList.js

```diff
@@ -16,6 +16,12 @@ function ButtonList(props) {
           message="Repeat Video"
           className="buttonListButton"
         />
+        <ToggleableButton
+          toggled={props.hideVideo}
+          setToggled={props.setHideVideo}
+          message="Hide Video"
```

Comment:
Can you make this toggleable also?
e.g.
```suggestion
          message={props.hideVideo ? "Show Video" : "Hide Video"}
```

Reply:
added